### DISCUSSION TITLE
How to test library locally?

### DISCUSSION THREAD

**Discussion by DedaDev** [js-fns/date-fns#2007] [Q&A]
How to test this library locally?
I have tried to include ./src/index.js in my node app but got a "using import statement outside of module" error, even though I have defined type="module" in package.json.

**Comment by michaelyuen** [comment]
Could you expand on what you mean by testing locally? I'm not sure importing from `src/index.js` makes sense to me. Linking (symlinking) is what I would think.

Another factor when it comes to node could be the version you're using. So what version are you using? `node -v`
